sites/docs/src/content/docs/tutorials/google_colab/nf-core-colab-guide.md

```diff
@@ -0,0 +1,178 @@
+```
+
+### Setting Up Conda for Google Colab
+
+Due to a lack of root access, it is not possible to run nf-core, or any Nextflow pipelines, using the `-profile docker` or `-profile singularity` container based configuration profiles.
+
+That leaves the `conda` profile as the only way to run pipelines.
+
+However, as Google Colab does not natively support conda, you need to install the [condacolab](https://pypi.org/project/condacolab/) package, which will set up Conda for you by running the following commands:
+
+```python
+!pip install -q condacolab
+import condacolab
+condacolab.install()
+```
+
+Make sure to set the conda defaults as follows for smooth pipeline execution:
+
+```python
+!conda config --add channels bioconda
+!conda config --add channels conda-forge
+!conda config --set channel_priority strict
+```
+
+## Running a test pipeline
+
+...and you're ready to go!
+
+Yes, the setup is that simple!
+
+Now that the main parts of the environment are all set up, we can run the demo pipeline [nf-core/demo](https://nf-co.re/demo/).
+To install the pipeline, run:
+
+```python
+! nextflow pull nf-core/demo
+```
+
+Now you can finally run your pipeline!
+
+```python
+! nextflow run nf-core/demo -profile conda,test --outdir demo-results
+```
+
+## Running and Editing Pipelines in VS Code via Colab
+
+While you could get away with editing existing pipelines inside Colab's built-in terminal using editors like vim or nano, a VS Code IDE offers a more robust and richer environment for development.
+
+If you use VS Code as your normal IDE, thankfully the [vscode-colab](https://github.com/EssenceSentry/vscode-colab) Python library provides just the toolkit you need to take advantage of Colab's hardware in the comfort of the popular VSCode software suite.
+
+It allows you to securely and reliably connect a local instance of VS Code to Google Colab as well as Kaggle notebooks through the use of the official [VS Code Remote Tunnels](https://code.visualstudio.com/docs/remote/tunnels) package.
+
+You can read more about the library and even help contribute to new features on its [GitHub repository](https://github.com/EssenceSentry/vscode-colab).
+
+:::note
+If you decide to use this approach, it is best to run this immediately after setting up the other dependencies that require installation by running inside Colab cells to avoid any connection issues.
+:::
+
+The first step is to install and import the library, which can easily be done by running the following command in your Colab code cell:
+
+```python title="Install vscode-colab"
+!pip install vscode-colab
+import vscode_colab
+```
+
+Next, authenticate the connection using your GitHub credentials:
+
+```python
+vscode_colab.login()
+```
+
+![interactive login popup in cell output](/images/tutorials/google_colab/login.png)
+
+Follow the displayed instructions to authorize the connection.
+
+To start the VS Code tunnel, optionally configure Git, set up a Python version, or create a new project:
```

Comment:
_To start the VS Code tunnel, optionally_ is somewhat contradictory, no? Either something is done to start the tunnel or it is optional and the tunnel also starts without. 

```suggestion
When starting the tunnel, you can configure some optional settings that may come handy for advanced use cases.
```
I would not mention e.g. the Python version. Only nf-core tools is written in Python, which runs on many current versions and the default Git configuration will probably do, no?